gvr-3dcursor/app/src/main/assets/settings.xml

```diff
@@ -226,11 +227,6 @@
             position="center"
             themeId="crystal_sphere"
             type="object">
-        <!--_VENDOR_TODO_ add your first device to the settings.xml here-->
```

Comment:
Why not just add the left_INDEX here, and add right_INDEX at line 256. Do we need to enable two other cursors for testing the IODevices. This will also need a note at 226 to change active=no to active=yes. Also names Left Cursor and Right Cursor are repeated which might be confusing in the settings UI.